doc/ProvisionReleaseBuild.md

```diff
@@ -38,32 +39,50 @@ sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config
    # Cortx Pre-requisites
    yum install --nogpgcheck -y java-1.8.0-openjdk-headless
    yum install --nogpgcheck -y python3 cortx-prereq sshpass
+   
    # Pre-reqs for Provisioner
    yum install --nogpgcheck -y python36-m2crypto salt salt-master salt-minion
+   
    # Provisioner API
    yum install --nogpgcheck -y python36-cortx-prvsnr
+   ```
+### 2. Verify provisioner version (0.36.0 and above)
+    provisioner --version
+   
+### 3. Create the config.ini file
+
+*Note:* You can find the devices on your node by running below command to update in config.ini
+    
+    device_list=$(lsblk -nd -o NAME -e 11|grep -v sda|sed 's|sd|/dev/sd|g'|paste -s -d, -)
 
-   # Cleanup temporary repos
-   rm -rf /etc/yum.repos.d/*3rd_party*.repo
-   rm -rf /etc/yum.repos.d/*cortx_iso*.repo
-   yum clean all
-   rm -rf /var/cache/yum/
-   rm -rf /etc/pip.conf
+  - Values for storage.cvg.0.metadata_devices:
+   ```
+    echo ${device_list%%,*}
+   ```
+  - Values for storage.cvg.0.data_devices:
+   ``` 
+    echo ${device_list#*,}
+   ```
+  - You can find the interfaces as per zones defined in your setup by running,
+   ```
+    firewall-cmd --get-active-zones
    ```
-### 4. Verify provisioner version (0.36.0 and above)
-   ```provisioner --version```
    
-### 5. Create the config.ini file
-   `vi ~/config.ini`
-   - Paste the code below into the config file replacing your network interface names with ens33,..ens37 and storage disks with /dev/sdc,/dev/sdb
+    vi ~/config.ini
+    
+   - Paste the code below into the config file replacing your network interface names with ens33,..ens37 and storage disks with /dev/sdb,../dev/sdf
    ```
    [srvnode_default]
    network.data.private_interfaces=ens34, ens35
    network.data.public_interfaces=ens36, ens37
    network.mgmt.interfaces=ens33
    bmc.user=None
    bmc.secret=None
-   storage.cvg.0.data_devices=/dev/sdc
+   
+   #data devices
+   storage.cvg.0.data_devices=/dev/sdc,/dev/sdd,/dev/sde,/dev/sdf
```

Comment:
Have we tested this configuration?

Reply:
yes

Reply:
changes to 2 disk as tested as well as ease for the community